applications/surgical_scene_recon/tests/test_minimal_holoviz.py

```
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
```

Ruff (0.14.4)

Shebang is present but file is not executable

(EXE001)